persist/pom.xml

```diff
@@ -124,7 +124,7 @@
             <schema>public</schema>
           </schemas>
           <encoding>ISO-8859-1</encoding>
-          <cleanOnValidationError>true</cleanOnValidationError>
+          <cleanDisabled>true</cleanDisabled>
```

Comment:
⛏️  This is a very important setting, can you add a comment here explaining this?